src/mlmc/tool/flow_mc.py

```diff
@@ -31,15 +30,15 @@ def create_corr_field(model='gauss', corr_length=0.125, dim=2, log=True):
         ])
 
     if model == 'exp':
-        model = gstools.Exponential(dim=dim, len_scale=len_scale)
+        model = gstools.Exponential(dim=dim, var=sigma**2, len_scale=len_scale)
```

Comment:
This is obsolete, `var` shouldn't be set in covariance model in our implementation.